ex6.4_remember.py

```diff
@@ -0,0 +1,19 @@
+from PIL import Image
+
+
+def decipher():
+    black = 1
+
+    with Image.open("code.png").convert() as img:
+        pixel = img.load()
+        w = img.size[0]
+        h = img.size[1]
+
+        return "".join([chr(j)
```

Comment:
Weird indentations, use pycharm's autoformat to fix your code - 
https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html